bescort.lic

```diff
@@ -1008,6 +1013,36 @@ class Bescort
       return
     end
   end
+
+def take_rh_lang_barge                                                                                                                                               
+  unless [50_948, 3434].include?(Room.current.id)                                                                                                                    
```

Comment:
Can you add room 466 to this list? There are some mapping issues and 50948 is a duplicate of 466.

Reply:
yeah I noticed that too, sometimes it's room 50_948 and other times it's 466. I'll just add 466 to the unless list, thanks.